yt_dlp/extractor/nuum.py

```diff
@@ -0,0 +1,200 @@
+import functools
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    OnDemandPagedList,
+    UserNotLive,
+    int_or_none,
+    parse_iso8601,
+    str_or_none,
+    url_or_none,
+)
+from ..utils.traversal import traverse_obj
+
+
+class NuumBaseIE(InfoExtractor):
+
+    def _call_api(self, path, video_id, description, query={}):
+        response = self._download_json(
+            f'https://nuum.ru/api/v2/{path}', video_id, query=query,
+            note=f'Downloading {description} metadata',
+            errnote=f'Unable to download {description} metadata')
+        if error := response.get('error'):
+            raise ExtractorError(f'API returned error: {error!r}')
+        return response.get('result')
```

Comment:
would be worse to return `None` than `KeyError` here
```suggestion
        return response['result']
```